test/models/tf_ov_model_zoo/run_infer_single.sh

```diff
@@ -57,83 +77,174 @@ function get_model_repo {
+    ret_code=$?
+    if (( $ret_code == 0 )); then
+        echo
+        echo "Stock Tensorflow: Checking inference result (warmups=$WARMUP_ITERS) ..."
+        ret_code=1
+        INFER_PATTERN=$( echo $INFER_PATTERN | sed -e 's/"/\\\\"/g' )
+        grep "${INFER_PATTERN}" ${TMPFILE} >/dev/null && echo "TEST PASSED" && ret_code=0
+        print_infer_times $NUM_ITER $WARMUP_ITERS "${TMPFILE}"
+        INFER_TIME_STOCKTF=$INFER_TIME
+    fi
+    echo
+    rm ${TMPFILE}
+    popd >/dev/null
+}
+
+function run_bench_stockov {
+    pushd . >/dev/null
+    VENVTMP="$WORKDIR/venv_temp_stockov" # to ensure no side-effects of any pip installs
+    [ -d $VENVTMP ] && rm -rf $VENVTMP
+    virtualenv -p python3 $VENVTMP
+    source $VENVTMP/bin/activate
+    pip list | grep 'opencv-python' 2>&1 >/dev/null; (($?==0)) || pip install opencv-python;
+    pip list | grep 'openvino' 2>&1 >/dev/null; (($?==0)) || pip install openvino;
+
+    cd ${LOCALSTORE}/demo
+    TMPFILE=${WORKDIR}/tmp_output$$
+    pythonlib=$(echo $(dirname $(which python3))/../lib)
+    echo LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$pythonlib ./run_ov_infer.sh ${MODEL} ${IMGFILE} $NUM_ITER $device
+    LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$pythonlib \
+        ./run_ov_infer.sh ${MODEL} ${IMGFILE} $NUM_ITER $device 2>&1 > ${TMPFILE}
+    ret_code=$?
+    if (( $ret_code == 0 )); then
+        echo
+        echo "Stock OpenVINO: Checking inference result (warmups=$WARMUP_ITERS) ..."
+        print_infer_times $NUM_ITER $WARMUP_ITERS "${TMPFILE}"
+        INFER_TIME_STOCKOV=$INFER_TIME
+    fi
+    echo
+    deactivate # venv
+    rm ${TMPFILE}
+    rm -rf $VENVTMP
+    popd >/dev/null
+}
+
 ################################################################################
 ################################################################################
 
-pip list | grep 'Pillow' 2>&1 >/dev/null; found=$(( ! $? ));
-if (( ! $found )); then pip install Pillow; fi
+pip list | grep 'Pillow' 2>&1 >/dev/null; (($?==0)) || pip install Pillow;
+if [ "${BENCHMARK}" == "YES" ]; then
+    pip list | grep 'networkx' 2>&1 >/dev/null; (($?==0)) || pip install networkx;
+    pip list | grep 'defusedxml' 2>&1 >/dev/null; (($?==0)) || pip install defusedxml;
+    pip list | grep 'test-generator' 2>&1 >/dev/null; (($?==0)) || pip install test-generator==0.1.1;
+fi
 
 cd ${LOCALSTORE_PREFIX} || exit 1
 get_model_repo
 
-TMPFILE=${LOCALSTORE_PREFIX}/tmp_output$$
-
 IMGFILE="${LOCALSTORE}/demo/images/${IMAGE}"
 if [ ! -f "${IMGFILE}" ]; then echo "Cannot find image ${IMGFILE} !"; exit 1; fi
-cd ${LOCALSTORE}/demo
-NUM_ITER=20
-[ -z "$NGRAPH_TF_LOG_PLACEMENT" ] && export NGRAPH_TF_LOG_PLACEMENT=1
-[ -z "$NGRAPH_TF_VLOG_LEVEL" ] && export NGRAPH_TF_VLOG_LEVEL=-1
 device=${NGRAPH_TF_BACKEND:-"CPU"}
+
+if [ "${BENCHMARK}" == "YES" ]; then
+    NUM_ITER=150
+    WARMUP_ITERS=50
+    export NGRAPH_TF_VLOG_LEVEL=-1
+else
+    NUM_ITER=20
+    WARMUP_ITERS=1
```

Comment:
1 and 0 may be enough here since we simply want to check if it works?

Reply:
This code is also used for non-benchmark pipeline, where we print average speed. That way we can check for regression on the average inference speed.

Reply:
The non-benchmark CI pipeline do not run exclusively and we shouldn't rely on them for average inference speed.

Reply:
Once kicked off we should monitor few runs and check the performacne.